src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -530,7 +484,7 @@ def prepare_or_run_pipeline(
         assert stack.container_registry
         image_name = deployment.pipeline.extra[ORCHESTRATOR_DOCKER_IMAGE_KEY]
         if self.is_local and stack.container_registry.config.is_local:
-            image_name = f"k3d-zenml-kubeflow-registry.{image_name}"
+            image_name = f"{stack.container_registry.name}.{image_name}"
```

Comment:
I'm not sure this is a safe way to actually solve this. This relies on users naming the registry exactly as output from the recipe right?